client.go

```diff
@@ -123,9 +122,15 @@ func (c *Client) pubMessages(in, out chan *Message, doneGen, donePub chan bool)
 		}
 	}
 
+	// Generate predictable client ID.
+	clientID := fmt.Sprintf("mqtt-benchmark-%v-%v", time.Now().Format(time.RFC3339Nano), c.ID)
```

Comment:
I don't think timestamp is really necessary in the client ID, this can be simplified to just making `mqtt-benchmark` default in main.go:

```go
clientPrefix   = flag.String("clientPrefix", "mqtt-benchmark", "MQTT client prefix prefix-id")
```

and then in `opts`:
```go
.SetClientID(fmt.Sprintf("%s-%v", c.ClientPrefix, c.ID))
```